imageloading-core/src/main/java/dev/chrisbanes/accompanist/imageloading/MaterialLoadingImage.kt

```diff
@@ -173,40 +190,47 @@ fun MaterialLoadingImage(
 }
 
 @Composable
-private fun fadeInColorFilter(durationMs: Int): ColorFilter {
+private fun updateFadeInTransition(key: Any, durationMs: Int): FadeInTransition {
     // Create our transition state, which allow us to control the state and target states
-    val transitionState = remember { MutableTransitionState(ImageLoadTransitionState.Empty) }
-    transitionState.targetState = ImageLoadTransitionState.Loaded
+    val transitionState = remember(key) {
+        MutableTransitionState(ImageLoadTransitionState.Empty).apply {
```

Comment:
Add a comment here for others to learn from e.g. 

`// Start with Empty state then immediately transition to Loaded`?

Reply:
That is actually done on purpose due to #141